src/pages/file-explorer-page.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useMemo } from "react";
+import files from "../components/FileExplorerTable/files.json"
+import { TableFileView } from "../components/FileExplorerTable";
+import { Icon } from "../components/Icon";
+
+
+export const FileExplorerPage = () => {
+
+  //Use state to transfer the selected file between the child/parent components
+  const [selectedFile, setSelectedFile] = useState("");
+
+  const changeFilePath = (file) => {
+    fetch(file)
+      .then(response => response.text())
+      .then(data => {
+        console.log(data);
+        setSelectedFile(data);
+      });
+  }
+
+  /*------ Section Helpers ------*/
+
+  function formatSize(numberString) {
+    if (numberString > 1024) {
+      numberString = numberString / 1024;
+      return numberString + "MB";
+    }
+    else {
+      return "0." + numberString + "MB";
+    }
+  }
+
+  function formatDate(dateString) {
```

Comment:
Will there be any sort of `dateString` edge cases that might cause issues here? Unless we're absolutely sure the `dateString` will **always** be in the format of YYYYMMDD this type of checking can be dangerous. Just making sure.